projects.html

```diff
@@ -109,7 +108,60 @@
-            <!-- End of Projects Section -->
+            <!-- End of students Projects Section -->
+              <section class="partproj-sec" id="partproj" href="#partproj">
+                <div class="row partproj-wrapper">
+                    <!-- Start ignoring BootLintBear -->
+                    <h2>Projects done by participants</h2>
+                    <!-- Stop ignoring -->
+                    {% for partproj in site.data.partproj %}
+                    <div class="col-xs-12 col-sm-6 col-md-4 col-lg-3">
+                        <div class="card-wrapper github-project" data-user="{{partproj.user}}" data-repo="{{partproj.repo}}">
+                            <div class="flip-container" ontouchstart="this.classList.toggle('hover');">
+                                <div class="flipper">
+                                    <div class="front">
+                                        <!-- front content -->
+                                        <div class="card stu-card">
+                                            <div class="avatar">
+                                                    <img src="images/partproj/{{partproj.image}}"
+                                                    alt="{{partproj.name}}"
+                                                    width="200px" height="200px">
+                                                    <p style="padding-top:3px;width: 200px;display: inline-block;margin-top: 9px;">
+                                                        <i class="fa fa-file-text"></i>
+                                                        &nbsp;
+                                                        <span class="stu-name card-text" title="{{partproj.name}}">
+                                                            {{partproj.name}}
+                                                        </span>
+                                                    </p>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="back">
+                                        <!-- back content -->
+                                        <div class="card stu-card">
+                                            <div class="avatar">
+                                                <p style="padding:10px;width: 200px;display: inline-block;margin-top: 9px;">
+                                                    <a style="text-decoration: none;" class="fa-2x" href="{{partproj.github}}" target="_tab">GitHub <i class="fa fa-external-link"></i></a>
+                                                </p>
+                                                <br><hr>
+                                                <center class="fa-2x">
+                                                    <i class="fa fa-code"></i> <span id="{{partproj.user}}-{{partproj.repo}}-language">HTML</span><br>
```

Comment:
we dont need this number of starts, while clicking on the link it should redirect to the gh pages of that project